client/state/ui/drop-zone/reducer.js

```diff
@@ -17,10 +13,16 @@ import {
 
 import { createReducer } from 'state/utils';
 
-const isVisible = createReducer( false,
+const isVisible = createReducer( {},
 	{
-		[ DROPZONE_SHOW ]: stubTrue,
-		[ DROPZONE_HIDE ]: stubFalse,
+		[ DROPZONE_SHOW ]: ( state, { dropZoneName } ) => ( {
+			...state,
+			[ dropZoneName ]: true,
+		} ),
+		[ DROPZONE_HIDE ]: ( state, { dropZoneName } ) => ( {
+			...state,
+			[ dropZoneName ]: false,
+		} ),
```

Comment:
Great suggestion! 

I thought that `{DE,}SERIALIZE` handling was needed for persistence, so if it doesn't support these events, it wouldn't be persisted anyway?